<launch>
    <arg name="events" default="true" />
    <arg name="sm_gui" default="true" />
    <arg name="reconf_gui" default="true" />

    <!-- task2 node -->
    <node name="task3" pkg="src_task3" type="val_task3_node" output="screen"/>

    <!-- state transition visualisation on rviz -->
    <node
    if="$(arg sm_gui)"
    name="rqt" pkg="rqt_gui" type="rqt_gui" args="-s rqt_decision_graph" output="screen" />

    <!-- Start up the dynamic reconfigure GUI. -->
    <node
    if="$(arg reconf_gui)"
    pkg="rqt_reconfigure" type="rqt_reconfigure" name="rqt_reconfigure"/>

    <!-- Start up rqt_console -->
    <!--node
    if="$(arg reconf_gui)"
    pkg="rqt_console" type="rqt_console" name="rqt_console"/-->

    <!-- launch the move group -->
    <include file="$(find valkyrie_moveit_config)/launch/move_group.launch" />

    <!-- start the robot fall detector -->
    <node pkg="navigation_common" type="fall_detector" name="fall_detector" />

</launch>

